nat/traversal/pinger.go

```diff
@@ -47,6 +48,7 @@ type Pinger struct {
 	portPool       PortSupplier
 	consumerPort   int
 	previousStage  string
+	publisher      Publisher
```

Comment:
Can we name it more clear? Just `publisher` does not provide info on what it does.

Reply:
But I was talking about field name, not the interface type. It fine to have `Publisher` as an interface name.

Reply:
So you're suggesting something like this?
```suggestion
	eventPublisher Publisher
```

Reply:
Correct.

Reply:
I actually dislike that variable name tells more than the interface, but whatever, changed it :D